packages/next/build/webpack/config/blocks/css/messages.ts

```diff
@@ -5,7 +5,7 @@ export function getGlobalImportError(file: string | null) {
     'cannot'
   )} be imported from files other than your ${chalk.bold(
     'Custom <App>'
-  )}. Please move all global CSS imports to ${chalk.cyan(
+  )}. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all global CSS imports to ${chalk.cyan(
```

Comment:
```suggestion
  )}. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party global CSS imports to ${chalk.cyan(
```